v0/destinations/blueshift/config.js

```diff
@@ -29,11 +29,57 @@ const EVENT_NAME_MAPPING = {
   Identify: "identify"
 };
 
+const BLUESHIFT_TRACK_EXCLUSION = [
+  "userId",
+  "id",
+  "email",
+  "event",
+  "type",
+  "token",
+  "idfa",
+  "advertisingId",
+  "idfv",
+  "id",
+  "manufacturer",
+  "carrier",
+  "ip",
+  "request_ip",
+  "latitude",
+  "longitude",
+  "messageId",
+  "cookie",
+  "device",
+  "os",
+  "network",
+  "address",
+  "traits"
```

Comment:
We don't need to exclude these properties here, as these are generally mapped by user